src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -38,29 +47,44 @@
 
 public class DefaultLoggerTest {
 
-    private final Violation auditStartMessage = new Violation(1,
-            Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditStarted", null, null,
-            getClass(), null);
+    private static final Locale DEFAULT_LOCALE = Locale.getDefault();
 
-    private final Violation auditFinishMessage = new Violation(1,
-            Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditFinished", null, null,
-            getClass(), null);
+    private final Object auditStartMessageClass;
+    private final Object auditFinishMessageClass;
+    private final Class<?> localizedMessage;
+    private final Constructor<?> cons;
+
+    public DefaultLoggerTest() throws Exception {
+        this.localizedMessage = getDefaultLoggerClass().getDeclaredClasses()[0];
+        this.cons = localizedMessage.getDeclaredConstructor(Object[].class, String.class);
+        cons.setAccessible(true);
```

Comment:
lets avoid this field `cons`, please inline it each method that requires it or create util method that create it a local variable.

reason is that test methods should be independent on each other to allow us in future run them in parallel, so only constant/immutable objects are ok to stay as fields.

Reply:
Since, we are using `newInstance`, we are creating a new class everytime which is independent of others. So, it is in a way util method only, where we have provided a data and get a new instance of class. Please let me know if I am wrong. 

Also, apologize for the delay. I do not have a system currently of my own, and so I am not able to fix issues quickly.

Reply:
you modify filed at https://github.com/checkstyle/checkstyle/pull/9904/files#diff-77d3e738400e9b22d5d6f52f535f2825c6c14a1f39e3389b8367546a4bc867f1R81
it will be very hard to debug problem when we come to Multithread execution of UTs.

we should avoid fields as much as possible. we should keep in fields only immutable objects.
We do not care about micro performance  degradation that might be cased by such object creation in each test method.

Reply:
Done